traitsui/qt4/check_list_editor.py

```diff
@@ -103,6 +86,42 @@ def list_updated(self, values):
 
         self.rebuild_editor()
 
+    def rebuild_editor(self):
+        """ Rebuilds the editor after its definition is modified.
+        Subclass should implement this method
+        """
+        raise NotImplementedError("rebuild_editor must be implemented.")
+
+# -------------------------------------------------------------------------
+#  'SimpleEditor' class:
+# -------------------------------------------------------------------------
+
+
+class SimpleEditor(BaseCheckListEditor):
+    """ Simple style of editor for checklists, which displays a combo box.
+    """
+
+    def init(self, parent):
+        """ Finishes initializing the editor by creating the underlying toolkit
+            widget.
+        """
+        self.create_control(parent)
+        super(SimpleEditor, self).init(parent)
+
+    def dispose(self):
+        """ Disposes of the contents of an editor.
+        """
+        if self.control is not None:
+            self.control.activated[int].disconnect(self.update_object)
+
+        super().dispose()
```

Comment:
Only added `dispose` for `SimpleEditor`, the `init` and `create_control` are the same as before.